client/src/routes/Search/Filters.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { PureComponent } from 'react'
+import Button from '../../components/atoms/Button'
+import styles from './Filters.module.scss'
+import data from '../../data/form-publish.json'
+
+const { steps } = data
+
+const labelCategories =
+    steps &&
+    steps[1].fields &&
+    steps[1].fields.categories &&
+    steps[1].fields.categories.label
+
+const optionsCategories =
+    steps &&
+    steps[1].fields &&
+    steps[1].fields.categories &&
+    steps[1].fields.categories.options
+
+const labelLicense =
+    steps &&
+    steps[2].fields &&
+    steps[2].fields.license &&
+    steps[2].fields.license.label
+
+const optionsLicense =
+    steps &&
+    steps[2].fields &&
+    steps[2].fields.license &&
+    steps[2].fields.license.options
+
+const filters = [
+    { label: labelCategories, items: optionsCategories },
+    { label: labelLicense, items: optionsLicense }
+]
+
+export default class Sidebar extends PureComponent<{
```

Comment:
```suggestion
export default class Filters extends PureComponent<{
```

whoops, should be `Filters`